hw_2.js

```diff
@@ -0,0 +1,110 @@
+// ex.1
+var a = 1, b = 1, c, d;
+c = ++a; alert(c);           // 2 т.к. операнд стоит до переменной, к "а" сразу прибавилось единца
+d = b++; alert(d);           // 1 т.к. операнд стоит после переменной "b", то сначала переменной d присвоили значения b. Затем прибавили единицу к b
+c = (2+ ++a); alert(c);      // 5 a = 3, следовательно 2+3=5
+d = (2+ b++); alert(d);      // 4 Перый шаг: 2+2=4 Затем к "b" прибавили единицу
+alert(a);                    // 3
+alert(b);                    // 3
+
+
+
+//ex.2 Чему будет равен x в примере ниже?
+var a = 2;
+var x = 1 + (a *= 2); // x = 5
+
+
+
+/* ex.3 Объявить две целочисленные переменные a и b и задать им произвольные начальные значения. Затем написать скрипт, который работает по следующему принципу:
+если a и b положительные, вывести их разность;
+если а и b отрицательные, вывести их произведение;
+если а и b разных знаков, вывести их сумму; ноль можно считать положительным числом. 
+*/
+var a = 3;
+var b = -3;
+
+if(a >= 0 && b >= 0 ) 
+   alert (a - b);
+else if (a < 0 && b < 0)
+   alert(x * y);
+else if ((a >= 0  && b < 0) || (a < 0 && b >= 0))
```

Comment:
Это условие не нужно
```suggestion
else {
```